### ISSUE TITLE
Filter Output Devices

### ISSUE THREAD

**Issue by limited660** [t3knomanzer/maxmix-software#186]
**Guidelines**
- [X] I have reviewed the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

**Is your feature request related to a problem? Please describe.**
Device currently shows all sound devices. Some of these are created for VR headsets and not used outside of Steam VR for example.

**Describe the solution you'd like**
Ability to choose which devices are shown in Output mode using software.

**Describe alternatives you've considered**
Devices can be removed from Output mode by disabling them in the control panel but this causes issues in other software that requires those devices.

**Additional context**
https://i.imgur.com/y4Mxlan.png

**Comment by limited660** [comment]
In addition, it would be nice to filter/whitelist/blacklist applications as well.

**Comment by Mavi222** [comment]
Yeah, choosing preferred applications is a must! I have plenty of processes running, and sometimes I can't find the game/teamspeak/discord in the game mode.
